orchestration/templates/process-and-reingest-release.yaml

```
@@ -459,3 +460,28 @@ spec:
     ## Inject template used to poll TDR jobs.
     {{- include "argo.poll-ingest-job" . | indent 4 }}
     {{- end }}
+
+      ##
+      ## Send a Slack notification describing the final status of the workflow.
+      ##
+    - name: send-slack-notification
+      container:
+        image: curlimages/curl:7.70.0
+        env:
+          - name: WORKFLOW_ID
+            value: {{ "{{workflow.name}}" | quote }}
+          - name: WORKFLOW_ENV
+            value: {{ .Values.jade.environment }}
+          - name: SLACK_URL
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.notification.slackUrl.secretName }}
+                key: {{ .Values.notification.slackUrl.secretKey }}
+        command: [curl]
+        args:
+        - -XPOST
+        - -H
+        - 'Content-type: application/json'
+        - -d
+        - '{"text": "Workflow $(WORKFLOW_ID) in env *$(WORKFLOW_ENV)* is cutting a new snapshot."}'
```

❓ It seems like this fires _after_ the snapshot is cut, given my reading of the `publish-processing-results` template. If that's the case, a minor rewording is in order, to get the tense right:
```suggestion
        - '{"text": "Workflow $(WORKFLOW_ID) in env *$(WORKFLOW_ENV)* has cut a new snapshot."}'
```